janusgraph-core/src/main/java/org/janusgraph/graphdb/util/AllEdgesIterable.java

```diff
@@ -1,92 +0,0 @@
-// Copyright 2017 JanusGraph Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package org.janusgraph.graphdb.util;
-
-import org.apache.tinkerpop.gremlin.structure.Edge;
-import org.apache.tinkerpop.gremlin.structure.Vertex;
-
-import java.util.Iterator;
-import java.util.Set;
-
-/**
- * Constructs {@link Iterable}s over all {@link org.janusgraph.core.JanusGraphEdge}s connecting a provided set of vertices.
- * <p>
- * Given a set of vertices, one may be interested in all edges that are contained in the subgraph spanned
- * by those vertices.
- *
- * @author Matthias Br&ouml;cheler (me@matthiasb.com);
- */
-public class AllEdgesIterable {
```

Comment:
I think the AllEdgesIterable and other classes under this package might be used by users directly.

Reply:
Should we mark them as deprecated in this release and maybe remove them in the next one?

Reply:
I think it is a good idea to deprecated them. They are not maintained over years, not tested and not documented.

Reply:
classes are deprecated